scout/utils/requests.py

```diff
@@ -142,22 +142,51 @@ def fetch_hpo_phenotype_to_terms():
 
     return fetch_resource(url)
 
-def fetch_ensembl_genes(build='37'):
-    """Fetch the ensembl genes
-
+def fetch_ensembl_biomart(attributes, filters, build=None):
+    """Fetch data from ensembl biomart
+    
     Args:
-        build(str): ['37', '38']
+        attributes(list): List of selected attributes
+        filters(dict): Select what filters to use
+        build(str): '37' or '38'
```

Comment:
But 38 doesn't work, did you understand why?

Reply:
Not really, I've googled a bit and followed some issues on GitHub. It seems like some problem with openssl has to do with it. Most probably its on the server side for Ensembl